core/src/main/java/org/apache/iceberg/rest/responses/ListTablesResponse.java

```diff
@@ -60,6 +69,7 @@ public static Builder builder() {
 
   public static class Builder {
     private final ImmutableList.Builder<TableIdentifier> identifiers = ImmutableList.builder();
+    private String nextPageToken = null;
```

Comment:
I think this also needs a builder method so that this token can actually be configured

Reply:
will add